helm/kanister-operator/templates/_helpers.tpl

```diff
@@ -24,6 +24,16 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*Create a default repository server controller name
+*/}}
+{{- define "repository-server-controller.name" -}}
+{{- if .Values.repoServerControllerName -}}
+{{- .Values.repoServerControllerName | trunc 63 | trimSuffix "-" -}}
```

Comment:
I cannot see this field `repoServerControllerName` being set in values.yaml. Do we need to add field with empty value here?

Reply:
Ah, that was missing, thank you